example/couchbase_example/client.cpp

```diff
@@ -1,20 +1,39 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
 #include <gflags/gflags.h>
 
 #include <chrono>
 #include <couchbase/codec/tao_json_serializer.hxx>
 #include <iomanip>
 #include <iostream>
+#include <tuple>
 #include <utility>
 #include <vector>
 
 #include "brpc/couchbase.h"
 
-DEFINE_string(couchbase_host, "couchbases://localhost",
-              "Couchbase server host");
-DEFINE_string(username, "selfdb", "Couchbase username");
-DEFINE_string(password, "Selfdb@1", "Couchbase password");
+DEFINE_string(couchbase_host, "couchbase://localhost", "Couchbase server host");
+DEFINE_string(username, "Administrator", "Couchbase username");
```

Comment:
Nitpick: I tend to not like to use Administrator/password everywhere, as it encourages bad security behavior.  They should have a normal user.  Not necessary to change though unless you think about it.